manifests/machineconfigdaemon/clusterrole.yaml

```diff
@@ -6,9 +6,15 @@ rules:
 - apiGroups: [""]
   resources: ["nodes"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["config.openshift.io"]
+  resources: ["clusterversions", "featuregates"]
+  verbs: ["*"]
```

Comment:
Hmm, do we need clusterversion access?

Also definitely read only, wildcard is not allowed anymore

Reply:
yes, FG requires clusterversions. I will just set to get, list